train_ta.py

```diff
@@ -0,0 +1,330 @@
+from pathlib import Path
```

Comment:
This is a mix of train.py from dvg and train.py from ai4eo, right?

Reply:
Sure more or less